src/app/core/shared/context.model.ts

```diff
@@ -16,4 +16,11 @@ export enum Context {
   AdminWorkflowSearch = 'adminWorkflowSearch',
   SideBarSearchModal = 'sideBarSearchModal',
   SideBarSearchModalCurrent = 'sideBarSearchModalCurrent',
+  MyDSpaceArchived = 'mydspaceArchived',
```

Comment:
honestly i'd prefer to create a badge context enum, instead of adding them here that represent the search context. Could move them in a new enum ?

Reply:
I see @atarix83 's point here now after reviewing the code more thoroughly.  These new `Context` settings are very different than the old ones.  All the new `MyDSpace*` Context values appear to map to *Badge* contexts.  While, the other Context values tend to represent an *Item* or search result.

However, that said, I do see the points that @artlowel makes about this design.

So, I'm fine with this as-is.  However, we might consider adding some comments in this `context.model.ts` to describe these different use cases.

For instance, just above this line, we could add something like:
```
// The MyDSpace* Context values below are used for badge display in MyDSpace.
```

(And perhaps add some similar comments to other Context values above about use cases...though I won't require it in this PR)